### PR TITLE
Metamask updater

### LINKED ISSUE #1: Migrate to Metamask 9.0.5

### LINKED ISSUE #2: Download Metamask instead of shipping bundled

### PR DESCRIPTION
**Pull request by BeroBurny**
closes #2 
closes #1 

implement dynamic metamask dowloading

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -3,5 +3,7 @@ node_modules
 test/server/data.js
 build
 dist
-yarn.lock
-.vscode/
\ No newline at end of file
+.vscode/
+*metamask
+.idea
+yarn-error.log
```
Comment (mpetrunic) [inline .gitignore:9]:
```suggestion
*.log
```

File: .gitignore [hunk 2]
```diff
@@ -3,5 +3,7 @@ node_modules
 test/server/data.js
 build
 dist
-yarn.lock
-.vscode/
\ No newline at end of file
+.vscode/
+*metamask
```
Comment (mpetrunic) [inline .gitignore:7]: I think you need to delete it first

File: package.json [hunk 3]
```diff
@@ -41,6 +42,7 @@
     "web3": "1.3.4"
   },
   "peerDependencies": {
-    "puppeteer": ">5"
+    "puppeteer": ">7",
```
Comment (mpetrunic) [inline package.json:45]:
```suggestion
    "puppeteer": ">5",
```

File: package.json [hunk 4]
```diff
@@ -41,6 +42,7 @@
...
+    "node-stream-zip": "^1.13.0"
```
Comment (mpetrunic) [inline package.json:46]: Ain0't this suppose to be prod dependency?

**Review by mpetrunic — COMMENTED** [review]
- merge master

File: package.json (hunk 4 above)
Comment (BeroBurny) [reply package.json:46]: :man_shrugging: we want to lock the user to a specific dependency?

File: package.json (hunk 3 above)
Comment (BeroBurny) [reply package.json:45]: it is updated to version 7, not sure it will work with a lower version of a puppeteer

File: package.json [hunk 5]
```diff
@@ -41,6 +42,7 @@
     "web3": "1.3.4"
   },
   "peerDependencies": {
-    "puppeteer": ">5"
+    "puppeteer": ">7",
+    "node-stream-zip": ">1.12.0"
```
Comment (mpetrunic) [inline package.json:46]: prod dependency

**Review by mpetrunic — APPROVED** [review]

**Pull request merged**